cclib/parser/turbomoleparser.py

```diff
@@ -46,10 +46,118 @@ def parse_basis(self, inputfile):
+
+        ## Atomic coordinates in job.last:
+        #              +--------------------------------------------------+
+        #              | Atomic coordinate, charge and isotop information |
+        #              +--------------------------------------------------+
+        #
+        #
+        #              atomic coordinates              atom shells charge pseudo isotop
+        #    -2.69176330   -0.00007129   -0.44712612    c      3    6.000    0     0
+        #    -1.69851645   -0.00007332    2.06488947    c      3    6.000    0     0
+        #     0.92683848   -0.00007460    2.49592179    c      3    6.000    0     0
+        #     2.69176331   -0.00007127    0.44712612    c      3    6.000    0     0
+        #     1.69851645   -0.00007331   -2.06488947    c      3    6.000    0     0
+        #...
+        #    -7.04373606    0.00092244    2.74543891    h      1    1.000    0     0
+        #    -9.36352819    0.00017229    0.07445322    h      1    1.000    0     0
+        #    -0.92683849   -0.00007461   -2.49592179    c      3    6.000    0     0
+        #    -1.65164853   -0.00009927   -4.45456858    h      1    1.000    0     0
+        if 'Atomic coordinate, charge and isotop information' in line:
+            self.skip_lines(inputfile,['border', 'b', 'b'])
+            line = next(inputfile)
+            if 'atomic coordinates' in line:
+                if not hasattr(self, 'atomcoords'):
+                    self.atomcoords = []
+
+                atomcoords = []
+                atomnos = []
+                line = next(inputfile)
+                while len(line) > 2:
+                    atomnos.append(line.split()[3].upper())
+                    atomcoords.append([utils.convertor(float(x), "bohr", "Angstrom") for x in line.split()[0:3]])
+                    line = next(inputfile)
+
+                self.set_attribute('atomcoords', atomcoords)
+                self.set_attribute('atomnos', [utils.PeriodicTable().number[atomelement] for atomelement in atomnos])
+
+        ## If we are unable to find coordinates in the job file, we will look for them
+        ## in the coord file.
+        #$coord
+        #   -2.69176330280845     -0.00007129445712     -0.44712612093731      c
+        #   -1.69851644615005     -0.00007332282157      2.06488947265450      c
+        #    0.92683848474542     -0.00007460039817      2.49592179180606      c
+        #    2.69176330586455     -0.00007127328394      0.44712611937145      c
+        #    1.69851645303760     -0.00007330575699     -2.06488946767951      c
+        #...
+        #   -7.04373605585274      0.00092243787879      2.74543890990978      h
+        #   -9.36352819434217      0.00017228707094      0.07445321997922      h
+        #   -0.92683848797451     -0.00007460625018     -2.49592178685491      c
+        #   -1.65164852640697     -0.00009927259342     -4.45456857763148      h
+        #$redundant
+        if line[0:6] == "$coord":

```

Comment:
In these situations, it's probably better to use `line.startswith` to avoid simple counting mistakes. Also more readable.